sources/deploy-options.sh

```diff
@@ -46,15 +60,15 @@ function deploy_options {
     echo
     echo "=================="
     echo
-    echo "---------------------"
-    echo "| APP STORE CONNECT |"
-    echo "---------------------"
+    echo "---------"
+    echo "| STORE |"
+    echo "---------"
     echo
-    echo "[3] App Store"
+    echo "[3] App/Play Store"
     echo
     read -r -p "Enter number in square brackets: " target_selection
```

Comment:
It might help to be a bit more descriptive and explain how you can put multiple numbers delimited by space